iis-workflows/iis-workflows-transformers/src/main/java/eu/dnetlib/iis/workflows/transformers/spark/citationmatching/direct/SparkCitationMatchingDirectTransformer.java

```diff
@@ -0,0 +1,64 @@
+package eu.dnetlib.iis.workflows.transformers.spark.citationmatching.direct;
+
+import java.io.IOException;
+
+import org.apache.spark.SparkConf;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+
+import eu.dnetlib.iis.citationmatching.direct.schemas.DocumentMetadata;
+import eu.dnetlib.iis.common.spark.avro.SparkAvroLoader;
+import eu.dnetlib.iis.common.spark.avro.SparkAvroSaver;
+import eu.dnetlib.iis.transformers.metadatamerger.schemas.ExtractedDocumentMetadataMergedWithOriginal;
+
+public class SparkCitationMatchingDirectTransformer {
```

Comment:
Considering that we have two implementations (one in spark and one in pig) then using 'spark' in class name let's us distinguish it right away. But since pig implementation is just a script and we don't have any class like 'PigCitationMatchingDirectTransformer' then maybe you're right.
I will leave it for now as is. But I will consider your comment when i will move transformer to citationmatiching-direct project